cfgov/v1/feeds.py

```diff
@@ -75,13 +56,15 @@ def get_appropriate_rss_feed_url_for_page(page, request=None):
     under the index page).
 
     Pages are considered to provide a feed if they inherit from
-    FilterableFeedPageMixin.
+    FilterableFeedRoutablePageMixin.
```

Comment:
Should this say `FilterableListMixin`?
```suggestion
    FilterableListMixin.
```